Allure.XUnit/AllureXunitPatcher.cs

```diff
@@ -1,106 +1,109 @@
 using System;
-
 using HarmonyLib;
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
-namespace Allure.XUnit
+namespace Allure.XUnit;
+
+internal static class AllureXunitPatcher
 {
-    static class AllureXunitPatcher
-    {
-        const string ALLURE_ID = "io.qameta.allure.xunit";
-        static bool isPatched = false;
-        static IRunnerLogger logger;
+    private const string ALLURE_ID = "io.qameta.allure.xunit";
+    private static bool _isPatched;
+    private static IRunnerLogger _logger;
 
-        static AllureMessageSink CurrentSink
+    private static AllureMessageSink CurrentSink
+    {
+        get
         {
-            get
+            var sink = AllureMessageSink.CurrentSink;
+
+            if (sink is null)
             {
-                var sink = AllureMessageSink.CurrentSink;
-                if (sink is null)
-                {
-                    logger.LogWarning("Unable to get current message sink.");
-                }
-                return sink;
+                _logger.LogWarning("Unable to get current message sink.");
             }
+
+            return sink;
         }
+    }
 
-        public static void PatchXunit(IRunnerLogger runnerLogger)
+    public static void PatchXunit(IRunnerLogger runnerLogger)
+    {
+        if (_isPatched)
         {
-            if (isPatched)
-            {
-                logger.LogMessage(
-                    "Patching is skipped: Xunit is already patched"
-                );
-                return;
-            }
+            _logger.LogMessage(
+                "Patching is skipped: Xunit is already patched"
+            );
 
-            logger = runnerLogger;
-            var patcher = new Harmony(ALLURE_ID);
-            PatchXunitTestRunnerCtors(patcher);
-            isPatched = true;
+            return;
         }
 
-        static void PatchXunitTestRunnerCtors(Harmony patcher)
+        _logger = runnerLogger;
+        var patcher = new Harmony(ALLURE_ID);
+        _patchXunitTestRunnerConstructors(patcher);
+        _isPatched = true;
+    }
+
+    private static void _patchXunitTestRunnerConstructors(Harmony patcher)
```

Comment:
This is an example of prefixing that bothers me.

```suggestion
    private static void PatchXunitTestRunnerConstructors(Harmony patcher)
```